components/components.yaml

```
@@ -724,3 +724,63 @@
   ProjectUrl: https://github.com/Particular/NServiceBus.TransactionalSession
   NugetOrder:
     - NServiceBus.TransactionalSession
+
+- Key: TransactionalSession.NHibernate
+  Name: NServiceBus.NHibernate.TransactionalSession
+  DocsUrl: /nservicebus/transactional-session/persistences/nhibernate
+  GitHubOwner: Particular
+  Category: Other
```

Should we use

```suggestion
  Category: Persistence
```

?